qiskit/circuit/tools/pi_check.py

```diff
@@ -104,9 +104,9 @@ def normalize(single_inpt):
             if power[0].shape[0]:
                 if output == "qasm":
                     if ndigits is None:
-                        str_out = "{}".format(single_inpt)
+                        str_out = f"{single_inpt}"
```

Comment:
Is there a reason to just do?
```suggestion
                        str_out = str(single_inpt)
```